goreleaser.yml

```diff
@@ -19,6 +19,9 @@ builds:
       - arm64
     goarm:
       - 7
+    ignore:
```

Comment:
I can't leave a comment on that line but should this be removed? https://github.com/aquasecurity/trivy/pull/646/files#diff-39e929907e3737ff7460d982cb6fbc60R41

Reply:
And this https://github.com/aquasecurity/trivy/pull/646/files#diff-39e929907e3737ff7460d982cb6fbc60R17

Or I guess in general all occurrences of 386?

Reply:
We should continue to support linux/386. This `ignore` section affects only darwin/386.

Reply:
Oh I see – I thought it was 386 in general.